GiniSwitchSDK/GiniSwitchSDK/Classes/ExtractionsManager.swift

```diff
@@ -319,7 +312,7 @@ class ExtractionsManager {
     }
     
     fileprivate func notifyExtractionsChanged() {
-        self.delegate?.extractionsManager(self, didChangeExtractions: extractions)
+        self.delegate?.extractionsManager(self, didChangeExtractions: extractions!)
```

Comment:
Are you 100% sure of this force unwrapping? You are using it in `parseExtractions` method, so you can either call the delegate directly there or pass the unwrapped extractions from there.